include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -20,7 +20,6 @@ namespace seqan3::detail
 
 /*!\brief The possible directions a trace can have. The values can be combined by the logical `|`-operator.
  * \ingroup alignment_matrix
```

Comment:
```suggestion
 * \ingroup alignment_matrix
 * \implements seqan3::enum_bitwise_operators
 * \sa seqan3::enum_bitwise_operators enables combining enum values.
```

`\implements` and `\sa` for all enums